homeassistant/components/tahoma/light.py

```diff
@@ -0,0 +1,133 @@
+"""Tahoma light platform that implements dimmable tahoma lights."""
+from datetime import timedelta
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_EFFECT,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_EFFECT,
+    LightEntity,
+)
+from homeassistant.const import STATE_OFF, STATE_ON, STATE_UNKNOWN
+
+from . import DOMAIN as TAHOMA_DOMAIN, TahomaDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=30)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the Tahoma light platform."""
+    if discovery_info is None:
+        return
+    controller = hass.data[TAHOMA_DOMAIN]["controller"]
+    devices = []
+
+    for device in hass.data[TAHOMA_DOMAIN]["devices"]["light"]:
+        devices.append(TahomaLight(device, controller))
+
+    async_add_entities(devices, True)
```

Comment:
```suggestion
    entities = []

    for device in hass.data[TAHOMA_DOMAIN]["devices"]["light"]:
        entities.append(TahomaLight(device, controller))

    async_add_entities(entities, True)
```

These are `entities`